tools/github_workflows/workflows_template.libsonnet

```diff
@@ -0,0 +1,141 @@
+{
```

Comment:
Why has this been copied? Would it be possible to remotely reference the workflow template we have in bb-storage?

Reply:
Purely an oversight on my part, removed now.